File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/recipes/rest/StepModifyRequest.java
package com.tarbadev.witchcraft.recipes.rest;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class StepModifyRequest {
  private Integer id;
  private String name;
}

File: WitchcraftApp/src/main/kotlin/com/tarbadev/witchcraft/recipes/rest/entity/StepModifyRequest.kt
package com.tarbadev.witchcraft.recipes.rest.entity

import com.tarbadev.witchcraft.recipes.domain.entity.Step

data class StepModifyRequest(
    var id: Int = 0,
    var name: String = ""
) {
  fun toStep(): Step {
    return Step(id, name)
  }
}
